waspc/cli/exe/Main.hs

```diff
@@ -138,6 +141,7 @@ printUsage =
         cmd   "    start db              Starts managed development database for you.",
         cmd   "    db <db-cmd> [args]    Executes a database command. Run 'wasp db' for more info.",
         cmd   "    clean                 Deletes all generated code and other cached artifacts.",
+        cmd   "    reset                 Deletes all generated code, all cached artifacts, and the node_modules directory.",
```

Comment:
I'd say they probably want to do `reset` by default (delete everything, including node_modules).

I imagine them running these commands when they want a clean start (i.e., remove everything that can be removed), that's how I use it. But could be wrong.

I suggest `clean` and `clean --all` (instead of hard).

Reply:
Ok, `clean` and `clean --all` also sounds good.

I would have `clean` be the desired default behaviour. If you think that is cleaning everything, then let's do that. In that case we maybe don't even need other `clean` version, just `clean` that cleans everything is fine? or would you have something like `clean --generated`?

So I am ok with any choice you make, be it just `clean` that cleans everything, or `clean` that cleans everything + `clean --generated` that cleans just dotwaspdir, or `clean` that cleans just dotwaspdir and then `clean --all` that cleans everything -> whatever you think makes the most sense at the moment. If I had to choose, I would probably go with the first option because of its simplicity, which is one hard `clean` and that is it.

Idea: What if our `clean` doesn't delete the whole node_modules/, but just wasp SDK from it?

Reply:
Can't remember what we decided to do with this one.

I think it was `wasp clean` (deletes `.wasp` and maybe `node_modules`) and `wasp clean --reset` (deletes `.wasp`, `node_modules`, `package-lock.json`, and maybe `migrations`).

Whoever ends up implementing it can decide as far as I'm concerned.

Reply:
I ended up going with just `clean`, which deletes .wasp and also node_modules.